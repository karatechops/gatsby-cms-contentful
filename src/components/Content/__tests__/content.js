import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Content from '../index';

describe('Content', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Content width="medium">This is my content</Content>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
