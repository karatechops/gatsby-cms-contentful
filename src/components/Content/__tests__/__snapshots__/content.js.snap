// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  margin: 0px;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 384px;
  padding-left: 24px;
  padding-right: 24px;
}

@media only screen and (max-width:768px) {
  .c0 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c0 {
    padding-left: 12px;
    padding-right: 12px;
  }
}

<div
  className="c0"
>
  This is my content
</div>
`;
